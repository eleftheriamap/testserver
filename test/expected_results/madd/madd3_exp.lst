
test/exp/test/gen/madd/madd3_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	5115ff5a 	sub	w26, w26, #0x57f
   4:	d162cb9c 	sub	x28, x28, #0x8b2, lsl #12
   8:	51369fde 	sub	w30, w30, #0xda7
   c:	9b1c7b5e 	madd	x30, x26, x28, x30
  10:	8a000000 	and	x0, x0, x0
