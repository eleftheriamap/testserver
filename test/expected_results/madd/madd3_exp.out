Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 00000000fffffa81
X27    = 0000000000000000
X28    = ffffffffff74e000
X29    = 0000000000000000
X30    = ff74e003fca4d259
PC     = 0000000000000010
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0x5115ff5a
0x00000004 : 0xd162cb9c
0x00000008 : 0x51369fde
0x0000000c : 0x9b1c7b5e
0x00000010 : 0x8a000000
